packages/chrome/src/elements/nav/NavItem.tsx

```diff
@@ -13,21 +13,13 @@ import { useNavContext } from '../../utils/useNavContext';
 import { useChromeContext } from '../../utils/useChromeContext';
 
 interface INavItemProps extends HTMLAttributes<any> {
-  /**
-   * Applies product-specific color palette
-   **/
+  /** Sets a product-specific color palette */
   product?: PRODUCT;
-  /**
-   * Indicate which item is current in the nav
-   **/
+  /** Adds the current item to the nav */
   isCurrent?: boolean;
-  /**
-   * Indicate that the item contains a product logo
-   */
+  /** Adds a product logo to the item */
   hasLogo?: boolean;
```

Comment:
This prop does not add a product logo to the nav item. It indicates that the item contains a product logo.

Reply:
Updated in latest push.